index.html

```diff
@@ -13,61 +13,71 @@
     <title>My Portofolio</title>
   </head>
   <body>
+    <!--the document header-->
+    <header id="id-header-content">
+      <div class="class-my-logo">My Logo</div>
+
+      <a href="#" class="class-header-link">hello</a>
+      <a href="#" class="class-header-link">portofolio</a>
+      <a href="#" class="class-header-link">about</a>
+      <a href="#" class="class-header-link">contact</a>
+      
+      <img src="./images/ic_menu.svg" alt="menu button" class="class-side-menu">
+
+    </header>
+
     <main id="id-main-content"><!--The main container: all elements are in-->
 
-      <!--the document header-->
-      <header id="id-header-content">
-        <div class="class-my-logo">My Logo</div>
-        <img src="./images/ic_menu.svg" alt="menu button" class="class-side-menu">
-      </header>
 
       <!--this section contain information about the author-->
       <section id="id-presentation-section">
-        <h2 class="class-my-name">Marc Raphael</h2>
-
-        <!--Information about the author-->
-        <p class="class-about-me">
-          Hello! I am a software developer! i can help you build a product, feature or website.
-          look through some of my work and experience! If you like what you see and have
-          a project you need coded, feel free to contact me!
-        </p>
+        
+        <div class="class-presentation">
+          <h2 class="class-my-name">Marc Raphael</h2>
+
+          <!--Information about the author-->
+          <p class="class-about-me">
+            Hello! I am a software developer! i can help you build a product, feature or website.
+            look through some of my work and experience! If you like what you see and have
+            a project you need coded, feel free to contact me!
+          </p>
+
+          <!--Here is the transition to the next section-->
+          <button type="button" class="class-start-see-project"> Start collaboration </button>
+        </div>
 
-        <!--Here is the transition to the next section-->
-        <button type="button" class="class-start-see-project"> Start collaboration </button>
+        <!--a simple div to make a background image easily and end this section-->
+        <div class="class-litle-background"></div>
       </section> 
 
-      <!--a simple div to make a background image easily and end this section-->
-      <div class="class-litle-background"></div>
-
       <!--this section contain information about projects-->
       <section id="id-works-section">
+        
+        <ul class="class-project-container">
 
-        <!--Introduction to the the project list-->
-        <h1 class="class-sections-titles">Projects</h1>
+          <!--Introduction to the the project list-->
+          <li class="class-sections-titles works-section-title">Projects</li>
```

Comment:
In the `Projects` section, kindly observe the following:

- [ ] The `Projects` text should read from top to bottom as observed in the Figma design.
- [ ] Next to the title taking 70% of the page's width, with no margin on the right, and aligned to the right there's a featured project.

_**Figma Design**_
![figma](https://user-images.githubusercontent.com/26856491/122377821-5468e200-cf5d-11eb-9324-bb2b5185f385.png)

_**Your Design**_
![design](https://user-images.githubusercontent.com/26856491/122377366-ede3c400-cf5c-11eb-83a3-2109263731a8.png)

_For other Regular (non-featured) projects_

- [ ] The first one is just a mockup image aligned to the left of the page with no margin on the page and it takes about 25% of the page's width.

- [ ] The second project shares the first one's row but is bigger in height (taking also part of the next row also) and looks just like the featured project but the description is above the picture; also takes about 25% of the page's width and is aligned completely to the right with no margin in the page.

- [ ] The third project is just a mockup image (like the first one) and is below the first project with no margin on top or left, this one takes 50% of the page's width.

- [ ] Lastly, there's a fourth project below the third one but starts in the center of the page and it takes about 30% of the page's width.